### PR TITLE
test: init install script tests

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

### CODE REVIEW

File: pkg/scripts_test/go.mod [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module sumologic_scripts_tests
+
+go 1.18
+
+require (
+	github.com/influxdata/influxdb v1.10.0
```
Comment (swiatekm-sumo) [inline pkg/scripts_test/go.mod:6]: Why is this in the direct dependencies?

File: pkg/scripts_test/install_test.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+		return exiterr.ExitCode(), nil
+	}
+
+	return 0, fmt.Errorf("cannot obtain exit code: %v", err)
+}
+
+func TestInstallation(t *testing.T) {
+	defer tearDown(t)
+
+	_, err := os.Stat("/usr/local/bin/otelcol-sumo")
```
Comment (swiatekm-sumo) [inline pkg/scripts_test/install_test.go:105]: Shouldn't we be running the whole thing in an chroot? As is, it requires root and modifies system directories, which seems dangerous.

[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply pkg/scripts_test/install_test.go:105]: to ensure that it is not created at beginning of test

File: pkg/scripts_test/go.mod (hunk 1 above)
Comment (sumo-drosiek) [reply pkg/scripts_test/go.mod:6]: good question

File: pkg/scripts_test/install_test.go (hunk 2 above)
Comment (sumo-drosiek) [reply pkg/scripts_test/install_test.go:105]: How do you think that should work. Should we prepare tree with all binary dependencies and then run test in chrooted environment?

Should we do that inside or outside of the go binary?

Comment (sumo-drosiek) [reply pkg/scripts_test/install_test.go:105]: ok, i did some research and making chroot environment manually seems to be counterproductive. I would leave it as it is, as test have to be run as root to make some impact on the system

Comment (sumo-drosiek) [reply pkg/scripts_test/install_test.go:105]: ref: https://github.com/SumoLogic/sumologic-otel-collector/blob/5e7d67f52075f688368bb3a83f6334fd00222edd/pkg/scripts_test/install_test.go#L129-L216

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**